java/advanced/APIDemo/app/src/main/java/com/google/android/gms/example/apidemo/DFPPPIDFragment.java

```diff
@@ -26,7 +26,6 @@
 import android.widget.Toast;
 import com.google.android.gms.ads.doubleclick.PublisherAdRequest;
 import com.google.android.gms.ads.doubleclick.PublisherAdView;
-import com.google.security.annotations.SuppressInsecureCipherModeCheckerNoReview;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
```

Comment:
[]()

Reply:
Hi @affan0078, please refrain from leaving empty and unhelpful reviews and comments on this repository. We welcome constructive input however consistent spammy interactions like this are not constructive and may result in you being blocked from the repo.